packages/velo-external-db-commons/lib/data_commons.js

```diff
@@ -65,10 +78,44 @@ const AdapterFunctions = {
     count: 'count'
 }
 
+const wixOperatorToAdapterOperator = operator => {
```

Comment:
Extract this function from `filter_transformer.js` because I need it also in tests and I didn't want to duplicate myself